types/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Minimum TypeScript Version: 3.0
+import {Plugin} from 'unified'
+
+/**
+ * Add `id`s to headings.
+ */
+declare const slug: Plugin
```

Comment:
The plugin accepts no options, plugin with the default generic accepts one option

```suggestion
declare const slug: Plugin<[]>
```

would be more accurate.

Reply:
Ah, I didn't notice the index signature in the implicit default of the generic. I fixed that with the latest commit.